src/core/mount-setup.c

```diff
@@ -408,6 +409,8 @@ int mount_setup(bool loaded_policy) {
         mkdir_label("/run/systemd", 0755);
         mkdir_label("/run/systemd/system", 0755);
         mkdir_label("/run/systemd/inaccessible", 0000);
+        touch_file("/run/systemd/inaccessible_file", false, USEC_INFINITY, 0, 0, 0000);
+        label_fix("/run/systemd/inaccessible_file", false, false);
```

Comment:
Could `inaccessible_file` be a character device like `/dev/null`? Major & Minor =1 & 3.